src/Entity/Education.php

```diff
@@ -15,21 +16,35 @@ class Education
     private ?int $id = null;
 
     #[ORM\Column(length: 255)]
+    #[Assert\NotBlank()]
     private ?string $school = null;
 
     #[ORM\Column(type: Types::DATE_MUTABLE)]
+    #[Assert\Expression(
+        "!this.getEndDate() ?: this.getEndDate() > this.getStartDate()",
+        message: 'La date de début doit être inférieure à la date de fin.',
+    )]
+    #[Assert\NotBlank()]
     private ?\DateTimeInterface $startDate = null;
 
     #[ORM\Column(type: Types::DATE_MUTABLE)]
+    #[Assert\LessThan('tomorrow')]
+    #[Assert\NotBlank()]
```

Comment:
on a pas dit qu'on la rendait optionnelle cette date ? le not blank est pas utilie non ?